package/gluon-wlan-encryption-psk/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Gluon wlan encrypt
+Encrypt wireless networks in a non-Freifunk setup.
+
+This package allows to encrypt 802.11s and infrastructure networks using a pre shared key (psk).
+
+This setup results in passwords being stored in plain-text. Site-configuration, Firmware-downloads and
```

Comment:
The pre-shared keys need to be kept secret. They are saved as plain text in the site.conf, the resulting firmware images and on the router. Thus access to these files and deployed routers needs to be restricted to authorized persons.

Reply:
I like yanosz version better and would like to make a minor suggestion:

This setup requires in passwords being stored in the firmware images, site-configuration and on the devices in clear-text. The security of the network depends on the protection of these artifacts against unauthorized access. Do not rely on the devices ssh passwords for security.